features/config-create.feature

```diff
@@ -119,6 +119,28 @@ Feature: Create a wp-config file
       define( 'AUTH_SALT',
       """
 
+  Scenario: Configure with invalid table prefix
+    Given an empty directory
+    And WP files
+
+    When I run `wp config create --skip-check --dbname=somedb --dbuser=someuser --dbpass=sompassword --dbprefix=""`
+    And STDERR should be:
+      """
+      Error: --dbprefix cannot be empty
+      """
+
+    When I run `wp config create --skip-check --dbname=somedb --dbuser=someuser --dbpass=sompassword --dbprefix=" "`
+    And STDERR should be:
+      """
+      Error: --dbprefix can only contain numbers, letters, and underscores.
+      """
+
+    When I run `wp config create --skip-check --dbname=somedb --dbuser=someuser --dbpass=sompassword --dbprefix="wp-"`
+    And STDERR should be:
+      """
+      Error: --dbprefix can only contain numbers, letters, and underscores.
+      """
```

Comment:
If you're expecting an error, you need to use `When I try ...`

Reply:
I changed it to `When i try` and readded the code check